### PR TITLE
infinite-scroller: nuke arrow function for iOS compat

### PR DESCRIPTION
**Pull request by paulirish**

### CODE REVIEW

**Comment by paulirish** [comment]
Proactive merge as people are hitting this in the wild.
cc @surma

**Comment by msurguy** [comment]
👍  thanks @paulirish !

**Comment by surma** [comment]
Thanks mate!

File: infinite-scroller/index.html [hunk 1]
```diff
@@ -172,7 +172,7 @@
   if(!node.children || node.children.length == 0)
     return 0;
   var childrenCount = Array.from(node.children).map(numDomNodes);
-  return node.children.length + childrenCount.reduce((p, c) => p + c, 0);
+  return node.children.length + childrenCount.reduce(function(p, c){ return p + c, 0;});
```
Comment (MartelliEnrico) [inline infinite-scroller/index.html:175]: The zero should be outside the function

Comment (oldergod) [inline infinite-scroller/index.html:175]: will - if merged- fix in #42

**Pull request merged**